### ISSUE TITLE
Add environmental variable TESTRAIL_SUITEID

### ISSUE THREAD

**Issue by DoctorStupidMD** [bahmutov/cypress-testrail-simple#11]
Our TestRail project contains multiple suites - after reading through the accompanying blog post, it doesn't seem that there's a way to pass in a value for `suite_id`. Any hope that this will be a possibility in the future?

Alternatively, have I overlooked a way to specify the `suite_id` in the bin command and CI build command?

**Comment by DoctorStupidMD** [comment]
@bahmutov My colleague and I would like to work on a PR for this issue, can you please grant us permissions? @evasilevsky

**Comment by evasilevsky** [comment]
@bahmutov , I'm working with @DoctorStupidMD .

**Comment by bahmutov** [comment]
Sure, you may and then open a pull request to this repo, no special permissions is needed

**Comment by evasilevsky** [comment]
We can't push new branches to make the PR in the first place. 
`ERROR: Permission to bahmutov/cypress-testrail-simple.git denied to evasilevsky.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.`

**Comment by bahmutov** [comment]
Eric, you would need to fork the repository so you have your own copy, then open a pull request from your changes to my repo

**Comment by evasilevsky** [comment]
Thank you!
Neither of us do any open source, but this change would definitely help our workflow.

**Comment by DoctorStupidMD** [comment]
@bahmutov Hi, I have a couple of changes ready on a branch that includes: 

- A GET request to retrieve the suite
- The environment variable to `testRailInfo`
- The `--suite` flag and alias to `args`
- A condition to check for `suiteId` in the `startRun` function

Running the bin command `npx testrail-start-run` will grab the newly-added `TESTRAIL_SUITEID` variable from my .bash_profile and perform the POST request successfully, posting the new run and mapped test cases to TestRail. Overriding with the '--suite' flag also works properly. All in all, it launches a new run from the command line.

I'm not exactly sure how to test this in a CI run with environment variables configured in AWS CodeBuild, so here's to hoping that works, too.

The issue I'm having is that the run isn't reporting test results (alternatively, it doesn't seem connected to running the actual tests). When running `npx testrail-close-run`, it throws `'TestRail run %d still has %d untested cases'` (all fall under `runInfo.untested_count`) and `'not closing the run...'` from /bin/testrail-close-run.js. Is this something we need to figure out/fix on our end or do you have the bandwidth to take it from there?

Thanks again for your engagement on the matter.

**Comment by slavab89** [comment]
Just a thought about this. Running a suit, if defined, will allow running only the suit. What if one would want to use parallel run of multiple suites - as part of a test plan. So maybe it would be possible to create a test plan (With multiple test suites inside), run the tests in parallel, and then close the plan

**Comment by DoctorStupidMD** [comment]
@slavab89 You bring up a good point - at least for our testing purposes, we need the suite id functionality added at the very least. I'm hoping that we can use this plugin with that addition soon, but perhaps @bahmutov would be open to another PR to address the case of multiple suites.

**Comment by DoctorStupidMD** [comment]
@bahmutov Any chance we could revisit this issue/PR?

**Comment by bahmutov** [comment]
Yeah I just don’t have time right now
